prow/github/client.go

```diff
@@ -1257,6 +1261,23 @@ func (c *client) authHeader() string {
 	return fmt.Sprintf("Bearer %s", token)
 }
 
+func (c *client) apiVersion() string {
+	resp, err := http.Get("https://api.github.com/versions")
+	if err != nil {
+		fmt.Print(err)
```

Comment:
Commenting just for the sake of completeness, see my verbose comment above otherwise about why I think we should not even have this check.

1. Errors should be always handled, not just written out. Sometimes the handling is "stop what you are doing and return the error so that something above in the chain handles it"
2. You basically never want to continue when something returns `err` - you must not rely the other returned value  (`resp` in this case).